kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/service/GwtPackageService.java

```diff
@@ -26,5 +26,7 @@
 	public List<GwtDeploymentPackage> findDeviceDeploymentPackages(GwtXSRFToken xsrfToken) throws GwtKuraException;
 	
 	public void uninstallDeploymentPackage(GwtXSRFToken xsrfToken, String packageName) throws GwtKuraException;
+	
+	public String getMarketplaceUri(GwtXSRFToken xsrfToken, String url) throws GwtKuraException;
```

Comment:
@dwoodard1 I don't see this method as very cohesive with the other packages-related methods. Maybe we can create a server-side servlet to manage the marketplace-related methods. But I think that this could come later in one of the next releases.